src/common.rs

```diff
@@ -6,6 +6,62 @@ use bson::{self, Bson};
 use std::collections::BTreeMap;
 use std::str::FromStr;
 
+/// Level of desired consistency and isolation properties of the data read.
+#[derive(Debug, Clone, Copy, PartialEq, Eq, PartialOrd, Ord, Hash)]
```

Comment:
Out of curiosity, what's the rationale for deriving ordering on `ReadConcern`? I don't really think of read concerns has having a natural order, and I'm not sure that anyone would be able to intuitively guess the ordering of a given pair of read concerns.